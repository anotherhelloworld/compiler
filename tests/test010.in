21.21+1
21.21aqw