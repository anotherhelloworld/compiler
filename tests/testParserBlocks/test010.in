var i: integer;
begin
    repeat
    ;;;;;
    i := 50;
    until (i > 10);
end.