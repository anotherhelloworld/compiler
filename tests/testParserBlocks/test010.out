begin
   until
         i
      >
         10
repeat
         i
      :=
         50
end
